src/containers/Home/HomeHero/index.tsx

```diff
@@ -20,22 +19,17 @@ const HomeHero: FC = () => {
         <div className="HomeHero__left">
           <div className="HomeHero__left-content-container">
             <h1 className="HomeHero__title">
-              <strong>Build</strong> software
-              <br />
-              <span className="HomeHero__earn-text">
-                <strong>Earn</strong>
-              </span>{' '}
-              digital currency
+              <strong>Open-source community for creators</strong>
```

Comment:
let's remove the `strong` tag here, and for the `HomeHero__title` class, lets add `font-weight: 800`